tutorons/tests/css/test_query.py

```diff
@@ -7,10 +7,97 @@
 import json
 from bs4 import BeautifulSoup
 from django.test import Client
-
+import tinycss
 
 logging.basicConfig(level=logging.INFO, format="%(message)s")
 
+class FetchExplanationsForMatthewsCSSSelectorTest(unittest.TestCase):
+    
+    def setUp(self):
+        self.client = Client()
+
+    def get_regions(self, document):
+        resp = self.client.post('/css/scan', data={'origin': 'www.test.com', 'document': document})
+        regions = json.loads(resp.content)
+        return regions
+
+    def test_single_css_selector(self):
```

Comment:
It would be great if you could add some verbs to the method names better explain what we expect to happen when we run these tests.

For example:
* "test_find_single_css_selector_in_stylesheet"
* "test_no_detect_selector_for_java_class_definition"